ui/intermittents-view/redux/reducers.js

```diff
@@ -0,0 +1,57 @@
+import moment from "moment";
+import { setDateRange } from "../helpers";
+
+export const fetchData = (name = "") => (state = { data: {}, message: "" }, action) => {
+  switch (action.type) {
+    case `FETCH_${name}_SUCCESS`:
```

Comment:
Neat way to do this!  :)

Reply:
I can't take credit, it's in the docs :) https://redux.js.org/recipes/structuring-reducers/reusing-reducer-logic

Reply:
Oh.. haha.  I guess I should read those docs or sometime...  :)